packages/react-link/src/components/Link/Link.types.ts

```diff
@@ -75,6 +73,11 @@ export interface ILinkProps
    */
   theme?: ITheme;
 
+  /**
+   * A component that should be used as the root element of the link returned from the Link component.
```

Comment:
Should this be "component or element" or "component or tag" to make it clear that native elements are okay too?